guide/how-it-works/introduction.md

```diff
@@ -8,11 +8,16 @@ permalink: /guide/how-it-works/
 main_classes: -no-top-padding
 ---
```

Comment:
```suggestion
image: https://bitcoin.design/assets/images/guide/how-it-works/how-it-works-preview.png
---
```
Need to reference the preview image here so it gets included in the HTML head. This is what social media sites use for link previews.